src/configs/getopts.c4m

```diff
@@ -340,6 +340,15 @@ archive targeted by the insertion command.
       field_to_set: "zip.inject_binary"
     }
 
+    flag_multi_arg zip_allowed_extensions {
```

Comment:
hmmm you think this will be needed to be customized that often hence requiring cli flag?

we have similar extension related configs already in chalk and they are exclusively configured via the config file:

https://github.com/crashappsec/chalk/blob/8ec46f7abaab712c5c47632d4682d3b03edfac1f/src/configs/chalk.c42spec#L2759-L2761

my vote is not to complicate the CLI with more nuanced flags unless they are absolutely necessary for the functionality of the command

Reply:
removed the flag